addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -781,14 +781,14 @@ export function getCursors(document) {
     ];
 }
 
-export function preserveCursor(document) {
+export function preserveCursor(document, normalize = true) {
```

Comment:
@dmo-odoo This is a little strange, I think we might want to discuss it in 3D.
In a few words, `setSelection` "normalizes" the selection by default, making it so that depending on whether we happen to call it after a command or not, the selection will be normalized or not. Since `preserveCursor` calls `setSelection` and some things in the sanitizer require it, we end up with a selection that is sometimes normalized, sometimes not...
I tried normalizing by default every time we sanitize but looking a the test results, in some cases it seem to me like we wouldn't want that (say I pasted "<i>test</i>" for instance).
Also, that normalization is a bit brutal sometimes, forcing a selection from `a<iframe/>[]b` to `a<iframe>[]</iframe>b`, or (relevant to this task) into a link.